<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QWidget" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>start</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="LabelWidget" native="true">
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>LifeCreator</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlWidget" native="true">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_7" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="buttonsWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>创建项目</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>加载项目</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="helpButton">
           <property name="text">
            <string>教程</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Animnia</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
